### PR TITLE
feat(ios): enables de, fr, and km localizations

### PR DESCRIPTION
**Pull request by jahorton**
While finalizing #4577, I noticed that we do have localization resources available for a few languages.  This enables them.

Also corrects a mistake in crowdin.yml and its effects 'til this point.

### CODE REVIEW

File: ios/engine/KMEI/KeymanEngine/Classes/de.lproj/ResourceInfoView.strings [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
+/* Class = "UILabel"; text = "Scan this code to load this keyboard on another device"; ObjectID = "z2O-MT-IoV"; */
+"z2O-MT-IoV.text" = "Scan this code to load this keyboard on another device";
```
Comment (jahorton) [inline ios/engine/KMEI/KeymanEngine/Classes/de.lproj/ResourceInfoView.strings:3]: Noting this...

File: ios/engine/KMEI/KeymanEngine/Classes/fr.lproj/ResourceInfoView.strings [hunk 2]
```diff
@@ -0,0 +1,3 @@
+
+/* Class = "UILabel"; text = "Scan this code to load this keyboard on another device"; ObjectID = "z2O-MT-IoV"; */
+"z2O-MT-IoV.text" = "Scan this code to load this keyboard on another device";
```
Comment (jahorton) [inline ios/engine/KMEI/KeymanEngine/Classes/fr.lproj/ResourceInfoView.strings:3]: and this, is it possible that our Crowdin upload tool has been missing the base localization file?  No one's localized it, despite covering most everything else.

Comment (darcywong00) [reply ios/engine/KMEI/KeymanEngine/Classes/fr.lproj/ResourceInfoView.strings]: Crowdin has the French string
https://translate.keyman.com/translate/keyman/448/en-fr?filter=basic&value=3

Comment (jahorton) [reply ios/engine/KMEI/KeymanEngine/Classes/fr.lproj/ResourceInfoView.strings]: Found the issue and nixed it already.  The .yml file had an error.

File: ios/engine/KMEI/KeymanEngine.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -520,6 +520,14 @@
...
 		CEA9670E24BEF05A0035AACF /* Updates.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = Updates.swift; sourceTree = "<group>"; };
+		CEACC90125F07C81006EAB45 /* de */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = de; path = de.lproj/ResourceInfoView.strings; sourceTree = "<group>"; };
+		CEACC90325F07CAF006EAB45 /* de */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = de; path = de.lproj/Localizable.strings; sourceTree = "<group>"; };
+		CEACC90425F07CB6006EAB45 /* km */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = km; path = km.lproj/Localizable.strings; sourceTree = "<group>"; };
+		CEACC90525F07CBA006EAB45 /* de */ = {isa = PBXFileReference; lastKnownFileType = text.plist.stringsdict; name = de; path = de.lproj/Localizable.stringsdict; sourceTree = "<group>"; };
+		CEACC90625F07CBC006EAB45 /* km */ = {isa = PBXFileReference; lastKnownFileType = text.plist.stringsdict; name = km; path = km.lproj/Localizable.stringsdict; sourceTree = "<group>"; };
```
Comment (darcywong00) [inline ios/engine/KMEI/KeymanEngine.xcodeproj/project.pbxproj:527]: ah, I found the 3rd "km" file was already at l.516

**Review by darcywong00 — APPROVED** [review]
lgtm

Maybe worth jotting a note in resources/build/l10n/README.md about any xcode special sauce you do when adding a language.

File: ios/engine/KMEI/KeymanEngine.xcodeproj/project.pbxproj (hunk 3 above)
Comment (jahorton) [reply ios/engine/KMEI/KeymanEngine.xcodeproj/project.pbxproj:527]: Yeah, there was a lingering 'km' inclusion artifact b/c it was the test language for the initial i18n work.

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 14.0.252-beta](https://keyman.com/downloads/releases/beta/14.0.252)

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.19-alpha](https://keyman.com/downloads/releases/alpha/15.0.19)

**Pull request merged**